libs/device-core/src/commands/use-cases/consent/reinstallConfiguration.ts

```diff
@@ -0,0 +1,75 @@
+import Transport, { StatusCodes, TransportStatusError } from "@ledgerhq/hw-transport";
+import { LocalTracer } from "@ledgerhq/logs";
+import { UserRefusedOnDevice, PinNotSet } from "@ledgerhq/errors";
+import type { APDU } from "../../entities/APDU";
+import type { ReinstallConfigArgs } from "../../entities/ReinstallConfigEntity";
+
+/**
+ * Name in documentation: REINSTALL_CONFIG
+ * cla: 0xe0
+ * ins: 0x6f
+ * p1: 0x00
+ * p2: 0x00
+ * data: CHUNK_LEN + CHUNK to configure at runtime
+ */
+const REINSTALL_CONFIG = [0xe0, 0x6f, 0x00, 0x00] as const;
+
+/**
+ * 0x9000: Success.
+ * 0xYYYY: already in REINSTALL mode
+ * 0xZZZZ: if other error (TBD)
+ */
+const RESPONSE_STATUS_SET: number[] = [
+  StatusCodes.OK,
+  StatusCodes.USER_REFUSED_ON_DEVICE,
+  StatusCodes.PIN_NOT_SET,
+];
+
+/**
+ * Requests consent from the user to allow reinstalling all the previous
+ * settings after an OS update.
+ *
+ * @param transport - The transport object used to communicate with the device.
+ * @returns A promise that resolves when the consent is granted.
+ */
+export async function reinstallConfiguration(
```

Comment:
[COULD] I know in the firmware doc it's called `REINSTALL_CONFIG` but in my opinion it's better to have something more explicit that describes what it actually does, like `reinstallConfigurationInit` or `requestReinstallConfigurationConsent`

Reply:
will do